modape/modis/download.py

```diff
@@ -183,8 +184,8 @@ def _parse_hdfxml(response):
             for datafile in entry.iter(tag="DataFiles"):
                 for datafilecont in datafile.iter(tag="DataFileContainer"):
                     for content in datafilecont:
-                        if content.tag in ["Checksum", "FileSize"]:
-                            result.update({content.tag: int(content.text)})
+                        if content.tag in ["FileSize", "ChecksumType", "Checksum"]:
```

Comment:
if easy to change, I’d prefer to use a tuple to hold values to check for. However, this might be a drop of water on a hot stone, as there might be multiple occasions where we use lists.